notebooks/responsibleaidashboard/responsibleaidashboard-housing-decision-making.ipynb

```diff
@@ -94,6 +95,17 @@
     "\n",
     "target_feature = 'SalePriceK'\n",
     "categorical_features = []\n",
+    "\n",
+    "outdirname = 'responsibleai.12.6.21'\n",
+    "try:\n",
+    "    from urllib import urlretrieve\n",
+    "except ImportError:\n",
+    "    from urllib.request import urlretrieve\n",
```

Comment:
just for background context, is this attempting to deal with different major versions of urllib with breaking api change?

Reply:
yep, exactly, different versions of urllib

Reply:
or actually, I think it was even more complicated than that. I recall installing pandas somehow even modifies this path.  I might be mis-remembering though.